pkgs/tools/security/tpm2-abrmd/default.nix

```diff
@@ -1,24 +1,36 @@
 { stdenv, fetchurl, lib
-, tpm2-tss, pkgconfig, glib, which, dbus, cmocka }:
+, makeWrapper, tpm2-tss, pkgconfig, glib, which, dbus, cmocka }:
 
 stdenv.mkDerivation rec {
   pname = "tpm2-abrmd";
-  version = "2.2.0";
+  version = "2.3.2";
 
   src = fetchurl {
     url = "https://github.com/tpm2-software/${pname}/releases/download/${version}/${pname}-${version}.tar.gz";
-    sha256 = "1lbfhyyh9k54r8s1h8ca2czxv4hg0yq984kdh3vqh3990aca0x9a";
+    sha256 = "040d01pdzkj0nc1c0vsf6gfqf28cgil03ix8dasijvhiha4c20nz";
   };
 
-  nativeBuildInputs = [ pkgconfig ];
+  nativeBuildInputs = [ pkgconfig makeWrapper ];
   buildInputs = [
     tpm2-tss glib which dbus cmocka
   ];
 
+  enableParallelBuilding = true;
+
   # Unit tests are currently broken as the check phase attempts to start a dbus daemon etc.
```

Comment:
Out of scope for you, probably, but it would be *really* cool to have a NixOS test that starts a VM with emulated TPM and tests everything through.

Reply:
Usually you wouldn't _need_ a VM as there is the IBM TPM simulator, though the tests of the `abrmd` assume a full system, not a limited build environment as forced by Nix. With enough hacks we could probably get this up and running, then again a full VM could to an integration test of the entire stack. In case I find myself with a bit of time on my hands, I can try to figure this out.